pyproject.toml

```diff
@@ -175,4 +176,8 @@ max-complexity = 30
 
 [tool.ruff.flake8-tidy-imports]
 [tool.ruff.flake8-tidy-imports.banned-api]
-"logging".msg = "The logging framework is banned. Please use demisto_sdk.commands.common.logger."
+# "logging".msg = "The logging framework is banned. Please use demisto_sdk.commands.common.logger."
+"json".msg = "use JSON_Handler instead"
+"ujson".msg = "use JSON_Handler instead"
+"orjson".msg = "use JSON_Handler instead"
+"ruamel.yaml".msg = "use YAML_Handler instead"
```

Comment:
why not just `ruamel` will the existing line prevent something like `from ruamel import yaml` ?

Reply:
ruff is smart enough to prevent those too.